src/preact/base-element.js

```diff
@@ -963,7 +963,9 @@ function parsePropDefs(props, propDefs, element, mediaQueryProps) {
   // as separate properties. Thus in a carousel the plain "children" are
   // slides, and the "arrowNext" children are passed via a "arrowNext"
   // property.
-  const nodes = element.getRealChildNodes ? element.getRealChildNodes() : [];
+  const nodes = element.getRealChildNodes
+    ? element.getRealChildNodes()
+    : element.childNodes;
```

Comment:
Can we assume that we don't have to filter out internal service nodes for non-custom elements?

Reply:
Done